_posts/2014-11-13-cockpit-plugin-tutorial.md

```diff
@@ -8,18 +8,18 @@ slug: creating-plugins-for-the-cockpit-user-interface
 
 *Note: This post has been updated for changes in Cockpit 0.90 and later.*
 
-[Cockpit is a user interface for servers](http://cockpit-project.org). And you can add stuff to that user interface. Cockpit is internally built of various components. Each component is HTML, with Javascript logic that makes it work, and CSS to make it pretty.
+[Cockpit is a user interface for servers](https://cockpit-project.org). And you can add stuff to that user interface. Cockpit is internally built of various components. Each component is HTML, with Javascript logic that makes it work, and CSS to make it pretty.
 
 It's real easy to create these components. Tools are components that show up in the *Tools* menu in Cockpit:
 
 ![Tools menu](/images/cockpit-tools-default.png)
 
 For example the *Terminal* that you see there is implemented as a tool. But lets make ourselves another one. For this tutorial you'll need Cockpit 0.41. You can install it in [Fedora 21](https://lists.fedorahosted.org/pipermail/cockpit-devel/2014-November/000196.html) or [build it from git](https://github.com/cockpit-project/cockpit/blob/master/HACKING.md).
 
-So break out your terminal, lets make a package called *pinger* that checks whether your server has network connectivity to the Internet by pinging another host. Nothing too fancy. We'll just be spawning a process on the server to do the work. I've prepared it for you as [an example here](http://cockpit-project.org/files/pinger.tgz), and we can look it over, and modify it. To download the example to your current directory:
+So break out your terminal, lets make a package called *pinger* that checks whether your server has network connectivity to the Internet by pinging another host. Nothing too fancy. We'll just be spawning a process on the server to do the work. I've prepared it for you as [an example here](https://cockpit-project.org/files/pinger.tgz), and we can look it over, and modify it. To download the example to your current directory:
 
 ```text
-$ wget http://cockpit-project.org/files/pinger.tgz -O - | tar -xzf -
+$ wget https://cockpit-project.org/files/pinger.tgz -O - | tar -xzf -
```

Comment:
Instructions to use `wget` (used a few times in website docs) should probably be checked.